index.html

```diff
@@ -13,6 +13,58 @@
   <title>Aliza Amir Portfolio</title>
 </head>
 <body>
+  <div class="popup-container hidden">
+    <div class="container-popup">
+        <div class="content">
+          <div class="pop-header-tech">
+            <div class="header-cross">
+          <h2 class="content-header margin-none">Tonic</h2>
+          <p class="cross-popup">X</p>
+        </div>
+          <ul class="specification margin-none margin-bottom">
+              <li class="name">Aliza</li>
+              <li class="skills"><img src="./images/dot.png" alt="">
+              <p> Back End Dev</p></li> 
+              <li class="year"><img src="./images/dot.png" alt="">
+                <p>2015</p></li>
+          </ul>
+        </div>
+          <div class="image">
+            <img class="img pop-image" src="./images/tonic.png" alt="">
+        </div>
+      </div>
+      <div class="para-links">
+        <div class="popup-paragraph">
+        <p class="content-para">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Commodi delectus tempora voluptate pariatur voluptas aut voluptatibus, alias eaque distinctio optio, at eligendi tenetur ab animi ut! r  </p>
+      </div>
+      <div class="links-buttons-underline">
+        <ul class="technologies pop-technoligies">
+            <li class="html technologies1">Html</li>
+            <li class="Css technologies1">Css</li>
+            <li class="javascript technologies1">Javascript</li>
+            <li class="pophidden hidden technologies1">Github</li>
+            <li class="pophidden  hidden technologies1">Ruby</li>
+            <li class="pophidden hidden technologies1">Bootstrap</li>
+        </ul>
+        <div class="underline"></div>
+        <div class="pop-button">
+          <button class="see-project pop-project"><a href="#">
+            <div class="live">
+            <span class="text">See Live</span>
+            <img src="./images/live-icon.svg" alt="logo" class="button-logo">
+          </div>
+          </a></button>
+          <div class="live">
+          <button class="see-project pop-project"><a href="#">
+            <span class="text">See Source</span>
+            <img src="./images/github-blue.svg" alt="logo" class="button-logo">
+          </div>
+          </a></button>
+        </div>
+      </div>
+    </div>
+  </div>
+  </div>
```

Comment:
- [ ] All the code of the modal window and the works section must be populated dynamically with JavaScript, kindly refactor your code to have a dynamic web page. 👇🏽 

![image](https://github.com/alizaamir123/mobile-version/assets/58642949/37126f78-07dd-438f-be39-6d6dab191648)